assets/js/lib/test-utils/factories/hosts.js

```diff
@@ -14,6 +14,15 @@ const slesSubscriptionIdentifierEnum = () =>
     'sle-ha',
   ]);
 
+const saptuneNotesIDs = [
```

Comment:
Really strange to have this as a constant.
I think we should have a factory to create a sapnote (this faker.number.int)` you have here.
and other to create a list of notes maybe

Reply:
Yeah fair point, this would be much more elegant, i will change it in the next commit :D